fs/layer/layer.go

```diff
@@ -59,6 +59,9 @@ const (
 	memoryCacheType                 = "memory"
 )
 
+// rootPath stores the filesystem root path used for Statfs operations
+var rootPath string
```

Comment:
A global variable should be avoided for this as NewResolver is possibly called multiple times with different root.